CONTRIBUTING.md

```diff
@@ -26,6 +27,99 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Contributing to Libpod
+
+This section describes how to start a contribution to Libpod.
+
+### Prepare your environment
+
+Read the [install documentation to see how to install dependencies](install.md) .
+
+The install documentation will illustrate the following steps::
+- install libs and tools
+- check installed versions
+- configure network
+- how to install libpod from sources
+
+### Fork and clone libpod
+
+First you need to fork this project on GitHub.
+
+First go to your `$GOPATH` `cd $GOPATH/src`.
+
+Create a path that correspond to your clone `mkdir -p github.com/<you>`.
+
+Clone your fork locally:
+```shell
+$ git clone git@github.com:<you>/libpod github.com/<you>
+$ cd github.com/<you>/libpod
+```
+
+### Deal with make
+
+Libpod use a Makefile to realize common action like building etc...
+
+You can list available actions by using:
+```shell
+$ make help
+Usage: make <target>
+ * 'binaries' - Build podman
+ * 'changelog' - Generate changelog
+ * 'clean' - Clean artifacts
+ * 'docs' - Generate documentation
+ * 'gofmt' - Verify the source code gofmt
+ * 'install' - Install binaries to system locations
+ * 'integration' - Execute integration tests
+ * 'libpodimage' - Build the libpod image
+ * 'lint' - Execute the source code linter
+ * 'local-cross' - Cross local compilation
+ * 'perftest' - Build perf tests
+ * 'podman' - Build with podman
+ * 'podman-remote' - Build with podman on remote environment
+ * 'podman-remote-darwin' - Build with podman on remote OSX environment
+ * 'run-perftest' - Build and run perf tests
+ * 'shell' - Run the built image and attach a shell
+ * 'test' - Run tests on built image
+ * 'testunit' - Run unittest on the built image
+ * 'varlink_generate' - Generate varlink
```

Comment:
Over time, this will deviate from reality and could confuse users.  Consider excluding or ellipsing  the actual content.  For example:

```
$ make help
Usage: make <target>
...output...
```